Makefile

```diff
@@ -101,16 +59,24 @@ LDLIBS  += -lm -lgcc -lc
 
 include $(BOLOS_SDK)/Makefile.glyphs
 
-APP_SOURCE_PATH += src
-SDK_SOURCE_PATH += lib_stusb lib_stusb_impl
+include $(BOLOS_SDK)/Makefile.common_app
 
-ifneq ($(TARGET_NAME),TARGET_STAX)
-SDK_SOURCE_PATH += lib_ux
-endif
+# Here custom flags can be added
+CUSTOM_APP_FLAGS = 0x000 # No custom flags needed, see appflags.h in SDK for flags definition
+APP_FLAGS = $(shell echo $$(( $(COMMON_APP_FLAGS) + $(CUSTOM_APP_FLAGS) )) )
 
-ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_STAX))
-    SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
-endif
+APP_LOAD_PARAMS += --appFlags $(APP_FLAGS)
+
+APP_LOAD_PARAMS += --curve secp256k1
+
+APP_LOAD_PARAMS += --path "44'"
+
+APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+
+DEFINES += $(COMMON_APP_DEFINES)
+
```

Comment:
we need to do something about `dep/%.d: %.c Makefile`
I'm wondering where those .d files are used. they don't appear as dependency in .o targets: https://github.com/LedgerHQ/ledger-secure-sdk/blob/master/Makefile.rules_generic#L74

Reply:
also dropped from master